tests/rptest/tests/timequery_test.py

```diff
@@ -376,7 +376,15 @@ def query_timestamp(self, ts, kcat_src, kcat_rr):
         self.logger.info(
             f"Time query {ts} expected offset {offset_src}, read-replica {offset_rr}"
         )
-        assert offset_src == offset_rr, f"Expected offset {offset_src}, got {offset_rr}"
+        matches = offset_src == offset_rr
+        if not matches:
+            self.logger.info(
+                f"src cluster record at {offset_src}: "
+                "{kcat_src.consume_one(self.topic_name, 0, offset_src)}")
+            self.logger.info(
+                f"rr cluster record at {offset_rr}: "
+                "{kcat_rr.consume_one(self.topic_name, 0, offset_rr)}")
```

Comment:
nit: these operations get lost in the strings, shouldn't they be out?

Reply:
Done